src/Catalyst.Core.Lib/Kernel/Kernel.cs

```diff
@@ -320,5 +320,17 @@ public Kernel Reset(bool reset)
 
             return this;
         }
+        
+        public Kernel Uninstall(bool shouldUninstall)
+        {
+            if (shouldUninstall)
+            {
+                Logger.Information("Uninstalling Catalyst Node");
+                
+                new FileSystem.FileSystem().GetCatalystDataDir().Delete(true);
```

Comment:
Not a issue with your code as I see this is in other places in the kernel but I will reference this issue for later so we use one instance that is resolved in Autofac instead of new instances of file system.

Reply:
Yeah I though that was unncecessary too, maybe make `FileSystem` static?